algviz/picture/anchor.py

```diff
@@ -0,0 +1,10 @@
+from enum import Enum
+
+class Anchor(Enum):
+    """Relative Positions on Picture"""
+    TOPLEFT = (0,0)
+    TOP = (0.5, 0)
+    LEFT = (0, 0.5)
+    CENTER = (0.5,0.5)
+    RIGHT = (1,0.5)
+    BOTTOM = (0.5,1)
```

Comment:
This looks incomplete

Reply:
this needs more commenting. We have no reference for what these units mean or what they would be used for.